applications/GeoMechanicsApplication/custom_conditions/U_Pw_normal_lysmer_absorbing_condition.cpp

```diff
@@ -219,160 +247,142 @@ void UPwLysmerAbsorbingCondition<TDim, TNumNodes>::GetFirstDerivativesVector(Vec
-    CalculateRotationMatrix(rotationMatrix, Geom);
+    BoundedMatrix<double, TDim, TDim> rotation_matrix;
+    CalculateRotationMatrix(rotation_matrix, rGeom);
 
     const int local_perpendicular_direction = TDim - 1;
 
     // calculate constant traction vector part
-    stiffnessConstants[0] = rVariables.G / rVariables.virtual_thickness;
-    stiffnessConstants[1] = rVariables.Ec / rVariables.virtual_thickness;
+    stiffness_constants[0] = rVariables.G / rVariables.virtual_thickness;
+    stiffness_constants[1] = rVariables.Ec / rVariables.virtual_thickness;
 
-    BoundedMatrix<double, TDim, TDim> localKMatrix = ZeroMatrix(TDim, TDim);
-    BoundedMatrix<double, TDim, TDim> auxLocalKMatrix = ZeroMatrix(TDim, TDim);
+    BoundedMatrix<double, TDim, TDim> local_k_matrix = ZeroMatrix(TDim, TDim);
+    BoundedMatrix<double, TDim, TDim> aux_local_k_matrix = ZeroMatrix(TDim, TDim);
 
     rVariables.KAbsMatrix = ZeroMatrix(TDim, TDim);
 
     for (unsigned int idim = 0; idim < TDim; ++idim) {
-        localKMatrix(idim, idim) = stiffnessConstants[0];
+        local_k_matrix(idim, idim) = stiffness_constants[0];
     }
-    localKMatrix(local_perpendicular_direction, local_perpendicular_direction) = stiffnessConstants[1];
+    local_k_matrix(local_perpendicular_direction, local_perpendicular_direction) = stiffness_constants[1];
 
-    auxLocalKMatrix = prod(localKMatrix, rotationMatrix);
-    rVariables.KAbsMatrix = prod(trans(rotationMatrix), auxLocalKMatrix);
+    aux_local_k_matrix = prod(local_k_matrix, rotation_matrix);
+    rVariables.KAbsMatrix = prod(trans(rotation_matrix), aux_local_k_matrix);
 
     for (unsigned int idim = 0; idim < TDim; ++idim) {
         rVariables.KAbsMatrix(idim, idim) = abs(rVariables.KAbsMatrix(idim, idim));
     }
 }
 
-/// <summary>
-/// Calculates the extrapolation matrix for neighbour elements. Values from integration points are extrapolated to the nodes
-/// </summary>
-/// <param name="NeighbourElement"></param>
-/// <returns></returns>
+
 template< unsigned int TDim, unsigned int TNumNodes >
-Matrix UPwLysmerAbsorbingCondition<TDim, TNumNodes >::CalculateExtrapolationMatrixNeighbour(const Element& NeighbourElement)
+Matrix UPwLysmerAbsorbingCondition<TDim, TNumNodes >::CalculateExtrapolationMatrixNeighbour(const Element& rNeighbourElement)
 {
-    const GeometryData::IntegrationMethod rIntegrationMethodNeighbour = NeighbourElement.GetIntegrationMethod();
-    const GeometryType& rNeighbourGeom = NeighbourElement.GetGeometry();
-    const IndexType rNumNodesNeighbour = rNeighbourGeom.size();
-    const IndexType NumGPointsNeighbour = rNeighbourGeom.IntegrationPointsNumber(rIntegrationMethodNeighbour);
+    const GeometryData::IntegrationMethod integration_method_neighbour = rNeighbourElement.GetIntegrationMethod();
+    const GeometryType& r_neighbour_geom = rNeighbourElement.GetGeometry();
+    const IndexType r_num_nodes_neighbour = r_neighbour_geom.size();
```

Comment:
Probably the `r_` prefix is here redundant (since it's not a reference):
```suggestion
    const IndexType num_nodes_neighbour = r_neighbour_geom.size();
```